Add support for installing on Linux Mint

fernandoacorreia opened the pull request:
Resolves error message about unsupported platform.

Linux Mint is a Linux distribution based on Ubuntu and Debian.
Starting with Mint 17 and continuing until 2016, every release of
Linux Mint will be built on Ubuntu 14.04 LTS.

GordonTheTurtle commented:
Please sign your commits following these rules:
https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work
The easiest way to do this is to amend the last commit:
~~~console
$ git clone -b "install-on-linux-mint" git@github.com:fernandoacorreia/docker.git somewhere
$ cd somewhere
$ git commit --amend -s --no-edit
$ git push -f
~~~

Ammending updates the existing PR. You **DO NOT** need to open a new one.

fernandoacorreia committed:
Add support for installing on Linux Mint

Resolves error message about unsupported platform.

Linux Mint is a Linux distribution based on Ubuntu and Debian.
Starting with Mint 17 and continuing until 2016, every release of
Linux Mint will be built on Ubuntu 14.04 LTS.

Signed-off-by: Fernando Correia <fernando@fernandocorreia.info>

cpuguy83 commented:
Pretty sure I've seen 2 or 3 of these now.

jfrazelle commented:
I think the solution in #15001 will also work here

jfrazelle commented:
this is a dup of https://github.com/docker/docker/pull/14910, closing, thanks for contributing though it would be awesome if you could help us test that PR or #15001

fernandoacorreia commented:
All right thanks for reviewing.
